stream-chat-android-compose/src/main/res/drawable/stream_compose_ic_edit.xml

```diff
@@ -2,11 +2,8 @@
     android:width="24dp"
     android:height="24dp"
     android:viewportWidth="24"
-    android:viewportHeight="24"
-    >
-    <path
-        android:fillColor="#000000"
-        android:fillType="evenOdd"
-        android:pathData="M21,21.0001H3V16.7571L16.435,3.3221C16.8255,2.9317 17.4585,2.9317 17.849,3.3221L20.678,6.1511C21.0684,6.5416 21.0684,7.1746 20.678,7.5651L9.243,19.0001H21V21.0001ZM5,19.0001H6.414L15.728,9.6861L14.314,8.2721L5,17.5861V19.0001ZM18.556,6.8581L17.142,8.2721L15.728,6.8581L17.142,5.4441L18.556,6.8581Z"
-        />
+    android:viewportHeight="24">
+  <path
+      android:pathData="M2.9985,17.25V21H6.7485L17.8085,9.94L14.0585,6.19L2.9985,17.25ZM5.9185,19H4.9985V18.08L14.0585,9.02L14.9785,9.94L5.9185,19ZM20.7085,5.63L18.3685,3.29C18.1685,3.09 17.9185,3 17.6585,3C17.3985,3 17.1485,3.1 16.9585,3.29L15.1285,5.12L18.8785,8.87L20.7085,7.04C21.0985,6.65 21.0985,6.02 20.7085,5.63Z"
+      android:fillColor="#72767E"/>
```

Comment:
#000000

Reply:
Both fixed.
Thank you, apparently my Resource Manager preview is not updating previews correctly so I had to sift by hand.